### DISCUSSION TITLE
Comparison with @tailwindcss/jit

### DISCUSSION THREAD

**Discussion by Nick-Mazuk** [windicss/vite-plugin-windicss#142] [Q&A]
In the readme, Windicss claims…

> It's FAST - 20~100x times faster than Tailwind on Vite

Guessing this is a comparison with normal Tailwind? If so, are there any speed comparisons with the new jit option for Tailwind?

**Comment by antfu** [comment]
I have little knowledge about benchmarking, but here I make a repo in case you want to try them out yourself
https://github.com/antfu/vite-windi-tailwind-jit-compare

**Reply by Nick-Mazuk** [reply]
Thanks!
